hellojavas.js

```diff
@@ -0,0 +1,5 @@
+document.addEventListener("DOMContentLoaded", ()=> {
+   document.addEventListener("click")
```

Comment:
here you want to add a callback function on what you want to do if the button is clicked, you also don't want to add an event listener to the document, you want to add it to your button.

`button. addEventListener('click', () => console.log('I'm being clicked!'))`